src/platform/site-wide/mega-menu/tests/my-health-link/mega-menu.my-health.cypress.spec.js

```diff
@@ -10,10 +10,10 @@ describe(manifest.appName, () => {
     ApiInitializer.initializeUserData.withDefaultUser();
     LandingPage.visitPage({ serviceProvider: CSP_IDS.ID_ME });
     cy.injectAxeThenAxeCheck();
-    cy.get('[data-e2e-id="my-healthe-vet-4"]')
+    cy.get('[data-e2e-id^="my-healthe-vet-"]')
       .should('be.visible')
       .and('have.text', 'My HealtheVet');
-    cy.get('[data-e2e-id="my-healthe-vet-4"]').should(
+    cy.get('[data-e2e-id^="my-healthe-vet"]').should(
```

Comment:
non-blocking questions:
- i can't find what element is using `data-e2e-id="my-healthe-vet-"` in this repo. is this attribute being used elsewhere?
- should we add a `-` to stay consistent with the other `data-e2e-id`s?